react/getting_started_with_react/keys_in_react.md

```diff
@@ -27,6 +27,10 @@ As long as `keys` remain consistent and unique, React can handle the DOM effecti
 
 ### Using keys
 
+<div class="lesson-note" markdown="1">
+Reminder: We will be using `props` here, and you will learn more about them in the next lesson. For now, you just need to know that `props` are arguments that are passed into components. 
```

Comment:
Small suggestion: the lesson note styling already highlights the text, not sure if we need the "Reminder" phrase
```suggestion
We will be using `props` here, and you will learn more about them in the next lesson. For now, you just need to know that `props` are arguments that are passed into components. 
```